psychopy/parallel/_inpoutx64.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
```

Comment:
So you're essentially duplication `_inpout32.py` here? If the contents of `PParallelInpOut32 ` and `PParallelInpOutx64` are identical, there should be no need to duplicate them just for the sake of different names.